config/routes.rb

```diff
@@ -91,6 +91,12 @@
     post "badges/award_badges", to: "badges#award_badges"
   end
 
+  namespace :stories, defaults: { format: "json" } do
+    resource :feed, only: [:show] do
+      get ":timeframe" => "feeds#show"
```

Comment:
This is a slight routing change from how things work now: `/top/week` becomes `/stories/feed/week`, and `/latest` becomes `/stories/feed/latest`